### PR TITLE
DMP-436 pre authentication landing screen

### PR DESCRIPTION
**Pull request by bpires2**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DMP-436

### Change description ###

Styling and functionality of pre-authentication landing page for internal and external users of DARTS

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export function initAppFn(envService: AppConfigService) {
     TranscriptionsComponent,
     LoginComponent,
   ],
-  imports: [BrowserModule, AppRoutingModule, ReactiveFormsModule, MatIconModule, HttpClientModule],
+  imports: [BrowserModule, AppRoutingModule, ReactiveFormsModule, MatIconModule, HttpClientModule, ReactiveFormsModule],
```
Comment (davet1985) [inline src/app/app.module.ts:38]: `ReactiveFormsModule` us in there twice

Comment (bpires2) [reply src/app/app.module.ts]: second instance of `ReactiveFormsModule` has been removed in latest commit

File: src/app/components/login/login.component.html [hunk 2]
```diff
@@ -1 +1,45 @@
...
+          <div id="sign-in-item-hint" class="govuk-hint govuk-radios__hint">
+            I have an account for DARTS through my organisation.
+          </div>
+        </div>
+      </div>
+    </fieldset>
+  </div>
+  <button class="govuk-button" data-module="govuk-button" type="submit">Continue</button>
+</form>
+<!-- <a href="/auth/login" class="govuk-button">Continue</a> -->
```
Comment (davet1985) [inline src/app/components/login/login.component.html:45]: Is this comment needed?

Comment (bpires2) [reply src/app/components/login/login.component.html]: Forgot to remove, has been removed now

**Review by davet1985 — APPROVED** [review]

**Pull request merged**